pkg/node/node.go

```diff
@@ -1050,10 +1072,26 @@ func (s *Service) NodeGetInfo(ctx context.Context, req *csi.NodeGetInfoRequest)
 							} else {
 								nvmetcpConnectCount = nvmetcpConnectCount + 1
 							}
+
+							svcID, err := strconv.ParseInt(target.TrsvcID, 10, 64)
+							if err != nil {
+								log.Errorf("DPU connect failed", err)
+							}
+
+							err = goopicsi.NVMeControllerConnect(12, target.Portal, target.TargetNqn, svcID)
```

Comment:
`NVMeControllerConnect` is not the right function to call here
one should use `ConnectToRemoteAndExpose` https://github.com/opiproject/goopicsi/blob/main/goopicsi.go#L25
since it need to connect to remote + create subsystem + create controller + create namespace : all in one operation

Reply:
Discussed about this and fix will be done